validation/checks/license_check.go

```diff
@@ -0,0 +1,84 @@
+package checks
+
+import (
+	"io"
+	"net/http"
+	"regexp"
+
+        "github.com/UCLALibrary/validation-service/validation/config"
+        "github.com/UCLALibrary/validation-service/validation/csv"
+)
+
+// Error messages
+var (
+	noProfileErr = "supplied profile cannot be nil"
```

Comment:
```suggestion
```

Reply:
Since there is already an errorMessage in this package that describes this error it may be beneficial to use that instead of creating a new one

Reply:
I noticed we were going to have this issue when I started my new validator ticket so I created a ticket to move the errors from the individual checks into an errors.go file in the `checks` package itself: https://uclalibrary.atlassian.net/browse/SERV-1231

I'm fine with letting this float until someone picks up that ticket and refactors all the errors out of the individual check files. I agree it makes sense to reuse rather than recreate, but we could punt until this other ticket is picked up.